userguide/tasks.rst

```diff
@@ -1006,9 +1006,9 @@ method is selected.
    +===========================+===========+================+=================================================================================================================+
    | Name                      | All       | string         | Enter a descriptive :guilabel:`Name` for the replication. Replication Task names must be unique.                |
    +---------------------------+-----------+----------------+-----------------------------------------------------------------------------------------------------------------+
-   | Direction                 | All       | drop-down menu | Direction of travel. *PUSH* sends snapshots to a destination system. *PULL* receives snapshots from a           |
-   |                           |           |                | destination system. Choosing *PULL* requires entering a snapshot :guilabel:`Naming Schema` to identify the      |
-   |                           |           |                | snapshots to replicate.                                                                                         |
+   | Direction                 | SSH, NCT, | drop-down menu | Direction of travel. *PUSH* sends snapshots to a :guilabel:`Target Dataset`. *PULL* receives snapshots from a   |
```

Comment:
This just seems awkward to have two sentences starting with PULL. With "remote system" mentioned, we don't need the link to SSH connections.  Maybe:
```
*PULL* replicates snapshots identified by the :guilabel:`Naming Schema` from a remote system.
```